quartz/build.ts

```diff
@@ -29,8 +31,11 @@ type BuildData = {
   toRebuild: Set<FilePath>
   toRemove: Set<FilePath>
   lastBuildMs: number
+  depGraphs: Record<string, DepGraph<FilePath>>
```

Comment:
```suggestion
  dependencies: Record<string, DepGraph<FilePath>>
```
nit: /s/depGraphs/dependencies

Reply:
If @jackyzha0 also agrees I'll make the change in a future PR

Reply:
Changed